reactos/base/system/services/lang/pl-PL.rc

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Needs to be referenced in the main .rc. Please test your changes -- you'd notice that this is missing.